sample2/counterapi-increment/run.csx

```diff
@@ -0,0 +1,38 @@
+#r "Microsoft.WindowsAzure.Storage"
+using System.Net;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log, CloudTable tableBinding, string countername, int add)
```

Comment:
Minor suggestion: add -> addValue, to be more explicit that it's not referring to the verb.